src/org/traccar/protocol/WatchProtocolDecoder.java

```diff
@@ -62,6 +64,36 @@ public WatchProtocolDecoder(Protocol protocol) {
             .expression("(.*)")                  // cell and wifi
             .compile();
 
+    private static final String STATE = "State";
+    private static final String ALARM = "Alarm";
+
+    private static final int STATUS_LOW_BATTERY_STATUS_BIT = 0;
```

Comment:
I don't see any good reason for having these constants.

Reply:
the numbers?